### PR TITLE
Release database emulator v4.0.0

### PR DESCRIPTION
**Pull request by IanWyszynski**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+* Modify RTDB emulator to publish triggers to the functions emulator
```
Comment (ryanpbrewster) [inline changelog.txt]: let's add a note about the default rules being open now

Comment (bkendall) [reply changelog.txt]: And, this is breaking, so do something like `*BREAKING*`

File: changelog.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 * **BREAKING** The CLI no longer supports being run in Node 6 environments.
+* **BREAKING** Modify RTDB emulator to publish triggers to the functions emulator
```
Comment (ryanpbrewster) [inline changelog.txt]: this is not a breaking change

File: changelog.txt [hunk 3]
```diff
@@ -1 +1,3 @@
...
+* RTDB Emulator comes up with open rules by default
```
Comment (ryanpbrewster) [inline changelog.txt]: this is a breaking change

**Pull request merged**